src/components/Auth/ResetPassword/ResetPassword.react.js

```diff
@@ -0,0 +1,219 @@
+import React, { Component } from 'react';
+import Paper from 'material-ui/Paper';
+import TextField from 'material-ui/TextField';
+import RaisedButton from 'material-ui/RaisedButton';
+// import $ from 'jquery';
+import './ResetPassword.css';
+import AppBar from 'material-ui/AppBar';
+import PasswordField from 'material-ui-password-field';
+import { Link } from 'react-router-dom';
+// import Dialog from 'material-ui/Dialog';
+// import FlatButton from 'material-ui/FlatButton';
+// import PropTypes from 'prop-types';
+// import {RadioButton, RadioButtonGroup} from 'material-ui/RadioButton';
```

Comment:
why this import is commented? This is also giving travis fail. Please correct it.